.github/workflows/aws_tfhe_gpu_4090_tests.yml

```diff
@@ -15,13 +15,15 @@ env:
 on:
   # Allows you to run this workflow manually from the Actions tab as an alternative.
   workflow_dispatch:
-  pull_request:
-    types: [ labeled ]
+  schedule:
+    # Nightly tests @ 1AM after each work day
+    - cron: "0 1 * * MON-FRI"
 
 jobs:
   cuda-tests-linux:
     name: CUDA tests (RTX 4090)
-    if: ${{ github.event_name == 'workflow_dispatch' || contains(github.event.label.name, '4090_test') }}
+    if: github.event_name != 'schedule' ||
+      (github.event_name == 'schedule' &&  github.repository == 'zama-ai/tfhe-rs')
```

Comment:
no more label to manually run 4090 tests ?

Reply:
You're right, we still need that.